### PR TITLE
Jpa code samples

### PR DESCRIPTION
**Pull request by tugrulaltun**

### CODE REVIEW

File: hazelcast-integration/hibernate-jpa-2ndlevel-cache/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+<h2>Prerequisites</h2>
+You should have installed Apache Maven(http://maven.apache.org/download.cgi).
+
+By default some dependencies added to project in "pom.xml" file as follows:
+```
+    <dependency>
+        <groupId>com.hazelcast</groupId>
+        <artifactId>hazelcast-hibernate3</artifactId>
+        <version>3.0-RC2</version>
```
Comment (bilalyasar) [inline hazelcast-integration/hibernate-jpa-2ndlevel-cache/README.md]: this should be ${hazelcast.version}

File: hazelcast-integration/hibernate-jpa-2ndlevel-cache/pom.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    <artifactId>hibernate-2ndlevel-cache</artifactId>
+    <name>Hibernate 2nd Level Cache</name>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-hibernate3</artifactId>
+            <version>3.0-RC2</version>
```
Comment (bilalyasar) [inline hazelcast-integration/hibernate-jpa-2ndlevel-cache/pom.xml]: same problem here, should be ${hazelcast.version}

File: hazelcast-integration/hibernate-jpa-2ndlevel-cache/pom.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    <parent>
+        <artifactId>hazelcast-integration</artifactId>
+        <groupId>com.hazelcast.samples</groupId>
+        <version>0.1-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>hibernate-2ndlevel-cache</artifactId>
```
Comment (bilalyasar) [inline hazelcast-integration/hibernate-jpa-2ndlevel-cache/pom.xml]: also we can change artifactId, we can include jpa somewhere in the name and articaftId.

File: hazelcast-integration/pom.xml [hunk 4]
```diff
@@ -24,6 +24,7 @@
         <module>spring-hibernate-2ndlevel-cache</module>
         <module>enterprise-session-replication</module>
         <module>spring-security</module>
+        <module>hibernate-jpa-2ndlevel-cache</module>
```
Comment (bilalyasar) [inline hazelcast-integration/pom.xml]: this module name should be same with artifact name of corresponding module. currently maven throws null pointer exception while compiling.

**Comment by bilalyasar** [comment]
-also can you make your sample logic user interactive. Like asking to user add,remove,list employee. maybe showing cache size can be good. (you can access hazelcast instance via [HazelcastAccessor](http://docs.hazelcast.org/docs/3.3/javadoc/com/hazelcast/hibernate/instance/HazelcastAccessor.html)  )
-and you can squash the commits.

**Comment by bilalyasar** [comment]
there are 17 commits, can you please squash them?

File: hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/applicationContext-hazelcast.xml [hunk 5]
```diff
@@ -17,9 +17,10 @@
 
 	        <hz:network port="${hz.network.port}" port-auto-increment="${hz.network.port.auto.increment}">
 	            <hz:join>
-	                <hz:tcp-ip enabled="${hz.tcp.ip.enabled}">
-	                    <hz:members>${hz.members}</hz:members>
-	                </hz:tcp-ip>
+			<hz:multicast enabled="false" multicast-group="224.2.2.3" multicast-port="54327"/>
```
Comment (bilalyasar) [inline hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/applicationContext-hazelcast.xml]: why do you change this file? seems this is not related to this pr.

Comment (tugrulaltun) [inline hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/applicationContext-hazelcast.xml]: there is an exception in spring-hibernate-2ndlevel-cache .where did i do wrong?

Comment (bilalyasar) [inline hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/applicationContext-hazelcast.xml]: you must change this file: https://github.com/hazelcast/hazelcast-code-samples/blob/master/hazelcast-integration/spring-hibernate-2ndlevel-cache/src/main/resources/hazelcast-config.properties
and these changes should be belong to another pr. (with hibernate 2nd level cache small fix)
